cluster/gce/util.sh

```diff
@@ -462,10 +462,32 @@ function detect-nodes() {
 #   KUBE_MASTER_IP
 function detect-master() {
   detect-project
-  KUBE_MASTER=${MASTER_NAME}
-  echo "Trying to find master named '${MASTER_NAME}'" >&2
+  if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
+    local server_name
+    for (( tp_num=1; tp_num<=${SCALEOUT_TP_COUNT}; tp_num++ ))
```

Comment:
it is not very clear why the loop is here. it will alway set for the last RP or TP.

Reply:
removed this loop and tested on scale-up, scale-out 1*1, 2*2, all works well.